apps/fluent-tester/src/FluentTester/TestComponents/Avatar/BasicAvatar.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { AvatarSize, AvatarColor, Avatar, AvatarPresence } from '@fluentui-react-native/experimental-avatar';
+import { Switch, View, Text, Picker, ColorValue } from 'react-native';
+import { satyaPhotoUrl, undefinedText } from './../PersonaCoin/styles';
+import { commonTestStyles as commonStyles } from '../Common/styles';
+import { useTheme } from '@fluentui-react-native/theme-types';
+import { AvatarTokens } from '@fluentui-react-native/experimental-avatar';
+
+type WithUndefined<T> = T | typeof undefinedText;
+
+const allSizes: WithUndefined<AvatarSize>[] = [
+  undefinedText,
+  'size8',
+  'size24',
+  'size32',
+  'size40',
+  'size48',
+  'size56',
+  'size72',
+  'size100',
+  'size120',
+];
+
+const allColors: WithUndefined<AvatarColor>[] = [
```

Comment:
Having a PlatformColor in this list would be cool, if it's straightforward -- if not, don't worry about it